pylot/control/mpc/trajectory.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+
+
+class Trajectory:
+    def __init__(self, t_list, s_list, x_list, y_list, k_list, vel_list,
+                 yaw_list, accel_list=None, steer_list=None):
+        self.t_list = list(t_list)  # Time [s]
+        self.s_list = list(s_list)  # Arc distance list [m]
+        self.x_list = list(x_list)  # X coordinate list [m]
+        self.y_list = list(y_list)  # Y coordinate list [m]
+        self.k_list = list(k_list)  # Curvature list [1/m]
+        self.vel_list = list(vel_list)  # Tangential velocity list [m/s]
+        self.yaw_list = list(self._normalize_yaw(yaw_list))  # Yaw list [rad]
```

Comment:
`_normalize_yaw` does not take a self.

Reply:
Moved to utils and separated as a helper function. (also moved trajectory there).